src/PKSim.Core/Services/PKSimDimensionFactory.cs

```diff
@@ -70,15 +71,15 @@ public override string MergedDimensionNameFor(IDimension sourceDimension)
          if (isConcentration(sourceDimension))
             return PKSimConstants.UI.Concentration;
 
+         if(isAmount(sourceDimension))
+            return PKSimConstants.UI.Amount;
+
          return base.MergedDimensionNameFor(sourceDimension);
       }
 
-      private bool isConcentration(IDimension dimension)
-      {
-         return
-            dimension == DimensionRepository.MassConcentration ||
-            dimension == DimensionRepository.MolarConcentration;
-      }
+      private bool isConcentration(IDimension dimension) => dimension.IsOneOf(DimensionRepository.MassConcentration, DimensionRepository.MolarConcentration);
+
+      private bool isAmount(IDimension dimension) => dimension.IsOneOf(DimensionRepository.Mass, DimensionRepository.Amount);
```

Comment:
This is just to get the merged dimension so that we can get amount and mass in chart